template/electron/index.js

```diff
@@ -1 +1,78 @@
-require('./main');
+// Import parts of electron to use
+const {app, BrowserWindow} = require('electron');
+const path = require('path')
+const url = require('url')
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is garbage collected.
+let mainWindow;
+
+// Keep a reference for dev mode
+let dev = false;
+if ( process.defaultApp || /[\\/]electron-prebuilt[\\/]/.test(process.execPath) || /[\\/]electron[\\/]/.test(process.execPath) ) {
```

Comment:
It is my understanding that [`electron-prebuilt` was renamed to just `electron` in npm](https://www.npmjs.com/package/electron#installation) quite a while ago (i.e. Compass v1.7 is the last artifact we created that included it, and 1.8+ which was used for the plugins workshop was Electron 1.6.1).

Is there a reason to support the older process path?